src/events/requests/ResolveRequestEventHandler.ts

```diff
@@ -17,7 +17,8 @@ export default class ResolveRequestEventHandler implements EventHandler {
 
 		TaskScheduler.clearMessageTasks( reaction.message );
 
-		if ( BotConfig.request.prepend_response_message == PrependResponseMessageType.WhenResolved ) {
+		if ( BotConfig.request.prepend_response_message == PrependResponseMessageType.WhenResolved
+		   && BotConfig.request.ignore_prepend_response_message_emoji !== reaction.emoji.name ) {
```

Comment:
(Minor) I personally pefer putting `&&` at the end of the last line

Reply:
I prefer it this way, and I think there are other places in the code where it's at the beginning of one line.

Reply:
I prefer it in the front as well, makes it clearer what the condition is imo